modules/opp-architecture-installation.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_module-type: PROCEDURE
+[id="opp-architecture-installation_{context}"]
+= Install {product-title}
+
+To install {product-title}, you must install five products in a specific order. Install {ocp} first. Then, install {rh-rhacm}. It does not matter what order you install the remaining products: {quay}, {rh-storage-essentials-first}, and {acs}. See the following detailed installation information for each product:
```

Comment:
Should we offer a more user friendly installation method in these docs as well?

Maybe this is just too early in the process but internally we have solutions using [openshift-plus policyset](https://github.com/stolostron/policy-collection/tree/main/policygenerator/policy-sets/community/openshift-plus) and [Helm-Charts](https://github.com/hybrid-cloud-patterns/multicluster-devsecops/tree/e26bf91c2ec926a810703be12a0dc96010530b5d/charts/hub/opp) for doing this deployment.

Asking our customers to do 5 separate product installs for a single bundle that they have purchased just doesn't feel right. I'm curious if we are open to exploring documentation for a deployment that will be easier on the user.

Reply:
Hi @ascerra  This is an item that we should discuss during the team meeting. I agree that a single deployment would be easier, seamless, and more receptive to customers and our internal teams. I will request a discussion as soon as possible.

Reply:
@ascerra  We discussed this in the latest team meeting and the answer is that we will not have a universal installer; we want people to use policy sets for the installation.